charts/aoi/templates/prometheus/prometheus.yaml

```diff
@@ -13,6 +13,7 @@ spec:
         - -httpListenAddr=:9090
         - -promscrape.config=/etc/prometheus/config_out/prometheus.env.yaml
         - -promscrape.config.strictParse=false
+        - -promscrape.maxScrapeSize=25165824 # 24 MiB # Default value 16777216
```

Comment:
Looks fine, but perhaps we should consider if the args should be configurable via helm values.

Reply:
Sounds like a good solution. But shouldn't we take that up later, after we have investigated further if there is more that needs to be adjusted?